Specs/DateTime/German/DateTimeModel.json

```diff
@@ -33,8 +33,8 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "Comment": "Currently not supported. 04 will be parsed as the year",
-    "NotSupported": "dotnet, javascript, python",
+    "Comment": "Should be removed, as 22.04 could be an (incorrectly) formatted date (22.04.) or time (22:04).",
+    "NotSupported": "dotNet, javascript, python",
```

Comment:
No need for this capitalization change. It just makes the review harder. Let's always keep it lowercase.